docs/mobile-apps/features/audio-capture.md

```diff
@@ -0,0 +1,81 @@
+---
+id: audio-capture
+title: Audio Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Audio Capture 
+
+Audio Capture is a functionality that gives you the ability to record the audio stream generated by your native mobile app (Android or iOS/iPadOS) during a live or automated real device test. 
+
+With this feature, the audio is recorded on our side and merged with the video recording, which you can access in your **Test Results**.
+
+## What You'll Need
+
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+* A native Android, iOS, or iPadOS mobile app that makes HTTP/HTTPS requests. We currently support the following:
+  * Android 10 and higher
+  * iOS 10+ up to iOS 15.3
+
+
+## Using Audio Capture on Real Devices
+
+To enable audio capture in your tests: 
```

Comment:
Audio Capture is lowercase here and capitalized in her parts of the doc. I would pick one for consistency. @aoleszyn

Reply:
Corrected.